listmonk.rb

```diff
@@ -37,9 +38,10 @@ def subscribers
   end
 
   def create_lists(lists)
-    monkconn.copy_data 'COPY lists (uuid, name, type, description) FROM STDIN', enco do
+    monkconn.copy_data 'COPY lists (id, uuid, name, type, description) FROM STDIN', enco do
       lists.each do |l|
         monkconn.put_copy_data [
+          l['project_id'],
```

Comment:
Following my suggestion in panoptes.project_lists, the reference should be to `id` here rather than `project_id`
```suggestion
          l['id'],
```